projects/storefrontapp-e2e-cypress/cypress/integration/regression/account-profile/update-profile.e2e-spec.ts

```diff
@@ -0,0 +1,73 @@
+const UPDATE_PROFILE_URL = '/my-account/update-profile';
+
+describe('Update profile', () => {
+  describe('when an anonymous user tries to access the page', () => {
+    it('should be redirected to the login page', () => {
+      cy.visit(UPDATE_PROFILE_URL);
+      cy.location('pathname').should('contain', '/login');
+    });
+  });
+
+  describe('when a user is logged in', () => {
+    before(() => {
+      cy.requireLoggedIn();
+    });
+
+    beforeEach(() => {
+      cy.visit(UPDATE_PROFILE_URL);
+    });
+
+    const newTitle = 'dr';
+    const newFirstName = 'N';
+    const newLastName = 'Z';
+
+    it('should be able to update its profile', () => {
+      // update the data
+      cy.get('cx-update-profile-form').within(() => {
+        cy.get('[formcontrolname="titleCode"]').select(newTitle);
+        cy.get('[formcontrolname="firstName"]')
+          .clear()
+          .type(newFirstName);
+        cy.get('[formcontrolname="lastName"]')
+          .clear()
+          .type(newLastName);
+        cy.get('button[type="submit"]').click();
+      });
+
+      // check for spinner while updating
+      cy.get('cx-spinner').should('exist');
```

Comment:
In my smoke test, the update happened fast enough for the spinner not to display.  While it is a good idea, maybe we should remove this assertion since we don't know if the processing speed will allow it to show.

Reply:
Ok, removed.